src/options.js

```diff
@@ -27,6 +27,13 @@ const exampleConfig = Object.assign({}, defaults, {
 function normalize(options) {
   const normalized = Object.assign({}, options || {});
 
+  if (
+    "filepath" in normalized &&
```

Comment:
We check for the existence of "filepath" in here so that allows the JS API to also send "filepath", should this also be added to readme ?